Air in wind/footer

AirInWind opened the pull request:
![image](https://github.com/user-attachments/assets/604a4d9d-2392-48ee-9d0c-0002a673b1a4)
![image](https://github.com/user-attachments/assets/a56b813f-19e7-4dde-9f0f-8a5962f89810)

- added footer

brandontran1220 commented on src/components/footer.tsx:
```diff
@@ -0,0 +1,80 @@
+              <p className="font-inter mb-1 pr-2 text-xl font-black">
+                Follow us on
+              </p>
+              <span className="rounded-sm border-3 [&>svg]:h-5 [&>svg]:w-5">
+                <svg
+                  xmlns="http://www.w3.org/2000/svg"
+                  fill="currentColor"
+                  viewBox="0 0 448 512"
+                >
+                  <path d="M100.3 448H7.4V148.9h92.9zM53.8 108.1C24.1 108.1 0 83.5 0 53.8a53.8 53.8 0 0 1 107.6 0c0 29.7-24.1 54.3-53.8 54.3zM447.9 448h-92.7V302.4c0-34.7-.7-79.2-48.3-79.2-48.3 0-55.7 37.7-55.7 76.7V448h-92.8V148.9h89.1v40.8h1.3c12.4-23.5 42.7-48.3 87.9-48.3 94 0 111.3 61.9 111.3 142.3V448z" />
+                </svg>
+              </span>
```
use the <Image> container from the next the library instead. You should be importing this image above. You could take a look at boardcard.tsx to see how i used it.

AirInWind commented:
![image](https://github.com/user-attachments/assets/66a34e0f-369c-4e01-a223-7a970c29b6e7)
![image](https://github.com/user-attachments/assets/319d9963-83b9-490b-bcd8-8473586a508c)
![image](https://github.com/user-attachments/assets/8b792746-b638-470f-a5bf-41d3fd2b2a62)

- added a quote to the middle of the footer that is chosen at random on page refresh
- added a link tree and push the link towards the top right
- the image now routes back to the home page

brandontran1220 reviewed (approved):
LGTM!
